tools/cloud-build/babysit_tests.py

```diff
@@ -280,12 +288,20 @@ def get_default_project():
     parser.add_argument("-r", type=int, default=1,
                         help="Number of retries, to disable retries set to 0, default is 1")
     args = parser.parse_args()
+
+    assert (args.sha is None) ^ (args.pr is None), "either --pr or --sha are required"
+    if args.pr is not None:
+        sha = get_pr(args.pr)["head"]["sha"][:7]
+    else:
+        sha = args.sha[:7]
```

Comment:
Technically the :7 should not be needed here or on L294 as it is handled in babysitter. I think best to only trim in one place. Makes sense to me to leave it long here and let it be trimmed by babysitter.